packages/babel-preset-env-standalone/src/available-plugins.js

```diff
@@ -2,7 +2,7 @@ import { availablePlugins, registerPlugin } from "@babel/standalone";
 
 const notIncludedPlugins = {
   "transform-new-target": require("@babel/plugin-transform-new-target"),
-  "proposal-json-strings": require("@babel/plugin-proposal-json-strings"),
+  "transform-json-strings": require("@babel/plugin-transform-json-strings"),
```

Comment:
@existentialism What is this object for?

Reply:
It's a mapping of packages _not_ included in `babel-standalone`.